### ISSUE TITLE
Add incoming/outgoing info in SMSDatum

### ISSUE THREAD

**Issue by haunthy** [predictive-technology-laboratory/sensus#187]
It should be helpful to add an indicator of whether this text message is an incoming/outgoing message. Right now, after anonymized, all numbers are encrypted.

**Comment by MatthewGerber** [comment]
All messages are outgoing from the sender's perspective and incoming from the recipient's perspective. What would such a flag add to this? Can you describe your use case?

**Comment by haunthy** [comment]
This is what it currently looks like(already de-identified all sensitive data):
{"$type":"SensusService.Probes.Communication.SmsDatum, SensusAndroid","FromNumber":"2018695353419165680345650xxxxxxxxxxxx","ToNumber":"681777973474564211377xxxxxxxxxxxxx","Message":"f94bbde566e9d1be836c7deaf17d2a32a01761fdace7555266b44255f9549fd9","Id":"cecb1c12-c7df-44d9-85c1-xxxxxxxxxxx","DeviceId":"xxxxxxxxxxxxx","Timestamp":"xxxxxxxxxxxxx","ProtocolId":"289f5a5d-fb7a-4b45-a60d-xxxxxxxxxxxx","Anonymized":true}

So basically, for the participant, we only know the FromNumber and ToNumber (both are de-identified, not sure which is the participant's number), but we don't know if this message is sent from this participant or received from someone else. Can we add a key-value pair , e.g. "State", and set "State" to be "IN" to indicate this is an incoming message to this participant. "OUT" to indicate this is an outgoing message from this participant?
